truss-chains/truss_chains/utils.py

```diff
@@ -131,33 +141,46 @@ def get_free_port() -> int:
         return port
 
 
+# NOTE(dynamic-chainlet-config): Predict URLs
+# for services that belong to a Chainlet are
+# auto-populated through dynamic config.
```

Comment:
I like that you add some explaining comments. If you move the comment into the function body, it has better coherence, e.g. if you move the function with a code refactoring tool, it will automatically stay together.